src/routes/docs/components/Explore.svelte

```diff
@@ -0,0 +1,97 @@
+<script lang="ts">
+  import HAccordion from '../../../components/HAccordion.svelte';
+  import ProviderIcon from '../../../lib/assets/icon-provider.svg';
+  import GatewayIcon from '../../../lib/assets/icon-gateway.svg';
+  import SsoIcon from '../../../lib/assets/icon-sso.svg';
+  import CoreIcon from '../../../lib/assets/icon-core.svg';
+
+  let sections: ('open' | 'closed')[] = ['open', 'open', 'closed', 'closed', 'closed'];
+
+  function handleToggled(event: CustomEvent<{ index: string; state: 'open' | 'closed' }>) {
+    const { index, state } = event.detail;
+    const newState = state === 'open' ? 'closed' : 'open';
+    sections.forEach((_state, objIndex, array) => {
+      if (objIndex.toString() !== index) {
+        array[objIndex] = newState;
+      }
+    });
+  }
+</script>
+
+<div
+  id="explore_section"
+  class="relative mt-f96 flex w-[100vw] min-w-full flex-row bg-navy sm:px-f64 sm:py-f80 lg:px-f128 lg:py-f96 vertical-lg:min-h-[550-px] vertical-lg:w-[100%]"
+>
+  <div id="explore_description" class="relative flex flex-row bg-navy pr-f12 text-teal vertical-lg:min-h-[550px]">
+    <div class="flex flex-col gap-f32">
+      <span class="h1 mt-f8 text-clip text-wrap font-title font-normal">Explore &amp; Build</span>
+
+      <div class="sm text-white sm:max-w-[515px] sm:pb-[100px] lg:pb-[200px]">
```

Comment:
```suggestion
      <div class="text-sm text-white sm:max-w-[515px] sm:pb-[100px] lg:pb-[200px]">
```
I take back my comment from earlier. The line height in the designs is larger that what's defined in the SG. By just setting the text size, the line height is left alone.